tests/sentry/deletions/test_tagkey.py

```diff
@@ -1,97 +0,0 @@
-from __future__ import absolute_import
-
-from sentry import tagstore
-from sentry.tagstore.models import EventTag
-from sentry.models import ScheduledDeletion
-from sentry.tasks.deletion import run_deletion
-from sentry.testutils import TestCase
-
-
-class DeleteTagKeyTest(TestCase):
```

Comment:
Why was this deleted instead of creating events?

Reply:
Sorry what do you mean? Since the TagKey model is part of the legacy tagstore and will be completely removed soon, I don't think the test for tagkey deletions is very relevant anymore.

Reply:
ok, understood